tcms/rpc/tests/test_testrun.py

```diff
@@ -568,6 +581,7 @@ def test_update_with_product(self):
         self.assertEqual(updated_test_run["build"], updated_build.pk)
         self.assertEqual(updated_test_run["summary"], updated_summary)
         self.assertEqual(updated_test_run["stop_date"], test_run.stop_date)
+        self.assertEqual(updated_test_run["planned_stop"], test_run.planned_stop)
```

Comment:
You don't have to add this field everywhere there Test.update is used! One test scenario which makes sure that the field exists in the results and the other one which validate the format/editability is enough. The other scenarios validate other behavior of the update method anyway.